packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -148,6 +147,8 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
             }
             key={ selectedIndex }
             aria-atomic={ true }
+            role='textbox'
+            aria-readonly='true'
```

Comment:
@jspurlin Still think we will be adding confusion by not just making Dropdown editable. Just letting you know about this line here, because if we did make dropdown editable, then we'd need to pipe that into this aria tag.